lib/modules/datasource/go/releases-goproxy.spec.ts

```diff
@@ -195,46 +205,50 @@ describe('modules/datasource/go/releases-goproxy', () => {
       });
     });
 
-    it('handles pipe fallback', async () => {
-      process.env.GOPROXY = `https://example.com|${baseUrl}`;
-
-      httpMock
-        .scope('https://example.com/github.com/google/btree')
-        .get('/@v/list')
-        .replyWithError('unknown');
-
-      httpMock
-        .scope(`${baseUrl}/github.com/google/btree`)
-        .get('/@v/list')
-        .reply(
-          200,
-          codeBlock`
+    it.each([true, false])(
```

Comment:
please use string template syntax

Reply:
maybe its better readable with multiple columns but you loose the type support